### PR TITLE
Update template, tick version to k, rerender

### PR DESCRIPTION
**Pull request by pmlandwehr**
Fixes https://github.com/conda-forge/openssl-feedstock/issues/11

### CODE REVIEW

**Comment by pmlandwehr** [comment]
This is a very minimally version tick (added a `python 3.6.*` dependency for `py36` builds mirroring others, and updated the hash). Notably I did *NOT*:
* Specify any version of vc for `win and py36`, so that will likely break.
* Drop out `py34`-specific build instructions. Shouldn't cause errors, but is untidy.

**Comment by jakirkham** [comment]
Needs ` - vc14    # [win and py36]` in features.

**Comment by jakirkham** [comment]
Also please pick a `perl` version and pin exactly (probably the latest version should be fine). Then re-render after doing this.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -50,10 +52,12 @@ test:
 
 about:
   home: http://www.openssl.org/
-  license: Apache-style
-  license_family: Apache
   license_file: LICENSE
+  license: Apache
```
Comment (jakirkham) [inline recipe/meta.yaml]: Not actually Apache hence the `-style` above. In fact, we should probably just say `OpenSSL`.

**Comment by jakirkham** [comment]
Added a "fixes" note in the OP. Hope that is ok. Should close out the related issue when this is merged.

**Comment by jakirkham** [comment]
Also please add this to `build`.

```yaml
  skip: true  # [win and py36]
```

**Comment by pmlandwehr** [comment]
Should we really skip the `win and py36` build? I would hope that with the fixed `vc` requirement it'll be fine.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -32,7 +34,7 @@ requirements:
     - python             # [win]
     # required for win32: https://wiki.openssl.org/index.php/Compilation_and_Installation#Windows
     - nasm               # [win32]
-    - perl
+    - perl ==5.18.2
```
Comment (jakirkham) [inline recipe/meta.yaml]: Will need to be at least 5.20.3.1. I don't think we ever had `perl` 5.18. Please re-render after correcting this.

**Comment by jakirkham** [comment]
> Should we really skip the `win and py36` build? I would hope that with the fixed `vc` requirement it'll be fine.

It is fine. It just means we build the same package twice on the AppVeyor matrix. So am suggesting we pick one (`py35` or `py36`) and skip the other one.

Python is a proxy here for Windows compiler runtime, which must be the same between packages that are linked (e.g. `pyyaml` and `yaml`). Both Python 3.5 and 3.6 use the same compiler runtime (`vc14`). So we only need to build with once against this runtime even though both Pythons use it. So am suggesting we skip either `py35` or `py36` on Windows.

File: recipe/meta.yaml (hunk 2 above)
Comment (pmlandwehr) [inline recipe/meta.yaml]: Ah k. OS X defaults to a fairly old perl apparently.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -32,7 +35,7 @@ requirements:
     - python             # [win]
     # required for win32: https://wiki.openssl.org/index.php/Compilation_and_Installation#Windows
     - nasm               # [win32]
-    - perl
+    - perl >=5.20.3.1
```
Comment (jakirkham) [inline recipe/meta.yaml]: We can just make this `==` and re-render. Otherwise we get all 3 `perl`s. Sorry for the headache with this stuff.

Comment (pmlandwehr) [inline recipe/meta.yaml]: Naw that's fine.

**Comment by jakirkham** [comment]
Could some other people please take a look? Thanks.

cc @conda-forge/openssl

**Comment by jjhelmus** [comment]
LGTM

**Comment by mingwandroid** [comment]
@pmlandwehr, why was the stricter perl pin done here?

**Comment by jakirkham** [comment]
@mingwandroid, please see issue ( https://github.com/conda-forge/conda-smithy/issues/415 ). Happy to discuss in that issue.

**Pull request merged**